scripts/blocks-utils.js

```diff
@@ -104,10 +108,22 @@ function observe(elementToObserve, callback, ...args) {
   observer.observe(elementToObserve);
 }
 
+function getOriginUrl() {
```

Comment:
These methods are no longer needed, can be removed

Reply:
IMO these method should be in place, as we might have to change how a url is calculated based on some logic as we get more APIS. 
so having a method won't impact any of the existing implementation